meter.go

```diff
@@ -53,6 +53,16 @@ func (m *Meter) Snapshot() Snapshot {
 	return m.snapshot
 }
 
+// Clear sets accumulator, total and rate to zero and last update to current time.
+func (m *Meter) Clear() {
```

Comment:
Nit: I'd call this "reset". You "reset" a meter, you "clear" a meter registry.

Reply:
Right